Factory/ZipkinTracerFactory.php

```diff
@@ -30,12 +32,8 @@ public function create(string $projectName, string $agentHost, string $agentPort
     {
         $tracer = new NoopTracer();
 
-        if (!dns_get_record($agentHost) && !filter_var($agentHost, FILTER_VALIDATE_IP)) {
-            $this->logger->warning(self::class . ': could not resolve agent host "' . $agentHost . '"');
-            return $tracer;
-        }
-
         try {
+            $this->agentHostResolver->resolveAgentHost($agentHost);
```

Comment:
hm, the Naming is maybe a little bit strange. While i saw the code in the core bundle it seamed obvious, but when reading it "in action" i would expect that there is a return valiue and something is done with the agenthost after resolving it... the `resolveAgentHost` function-name doesn't sound like a check, but it is?

Reply:
True, I will rename it to `ensureAgentHostIsResolvable()` to make it more obvious.